packages/devtools_app/lib/src/shared/http/http_request_data.dart

```diff
@@ -229,6 +248,7 @@ class DartIOHttpRequestData extends NetworkRequest {
 
   /// Merges the information from another [HttpRequestData] into this instance.
   void merge(DartIOHttpRequestData data) {
+    _updateCount.value++;
```

Comment:
You might want to do this update after setting `_request` to make sure we don't trigger a rebuild until the data is updated.